Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.5
```

Comment:
This change could be disruptive for consumers.  I'm ok with it but it needs to be more than a patch version bump.  Perhaps a feature or major version, but I'm not sure.

Something else we could do is create another project/package that is just for documentation and is not shipped to consumers.  Since all this documentation is for developers working on SwiftCurrent, we don't have to ship the project.  That project would build all things being documented (probably one reason the liblint fails for CocoaPods when adding more files from SwiftCurrent_SwiftUI)

Additional thought, I wonder if a dedicated documentation project would allow us to not alias every file we are documenting.

Reply:
We spoke about this over a call, but the short version is that we do not want to create a separate project for consumers.

This change is *not* disruptive to consumers because of the other package.swift file you said we needed to delete...that's why we have 2 now.